exercises/practice/accumulate/Accumulate.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-function accumulate(array $input, callable $accumulator)
+function accumulate(array $input, callable $accumulator): array
 {
-    // YOUR CODE GOES HERE
+    \BadFunctionCallException("Implement the accumulate function");
```

Comment:
Shouldn't these throw the error?
```suggestion
    throw new \BadFunctionCallException("Implement the accumulate function");
```

Reply:
Holy... your are completely right. Fixed for all the tests.